### DISCUSSION TITLE
Error during following Human-GEM guide: Step 3 (Check tasks) issue

### DISCUSSION THREAD

**Discussion by sangmok920622** [SysBioChalmers/Human-GEM#879] [Q&A]
Hello, I have tried following the Human-GEM guide for making context-specific GEM by integrating bulk RNA-seq results.
However, I continuously have met the error during 'prepHumanModelForftINIT' function especially Step 3.
When used 'metabolicTasks_Essential.txt' as a metabolic task for making 'prepData', it didn't go further process after 'PASS: [SU] Beta oxidation of long-chain FA'.
I will attach my code status as follows:

```matlab
>> load('D:\Human-GEM\model\Human-GEM.mat')
>> prepData = prepHumanModelForftINIT(ihuman, false, 'D:\Human-GEM\data\metabolicTasks\metabolicTasks_Essential.txt', 'D:\Human-GEM\model\reactions.tsv');
Step 1: Gene rules
Step 2: First simplification


Step 3: Check tasks (~10 min)
PASS: [ER] Aerobic rephosphorylation of ATP from glucose
PASS: [ER] Aerobic rephosphorylation of GTP
PASS: [ER] Aerobic rephosphorylation of CTP
PASS: [ER] Aerobic rephosphorylation of UTP
PASS: [BS] ATP de novo synthesis
PASS: [BS] CTP de novo synthesis
PASS: [BS] GTP de novo synthesis
PASS: [BS] UTP de novo synthesis
PASS: [BS] dATP de novo synthesis
PASS: [BS] dCTP de novo synthesis
PASS: [BS] dGTP de novo synthesis
PASS: [BS] dTTP de novo synthesis
PASS: [SU] Histidine uptake
PASS: [SU] Isoleucine uptake
PASS: [SU] Leucine uptake
PASS: [SU] Lysine uptake
PASS: [SU] Methionine uptake
PASS: [SU] Phenylalanine uptake
PASS: [SU] Threonine uptake
PASS: [SU] Tryptophan uptake
PASS: [SU] Valine uptake
PASS: [IC] Glycerate 3-phosphate  de novo synthesis
PASS: [IC] Mitochondrial acetyl-CoA de novo synthesis
PASS: [IC] Mitochondrial AKG de novo synthesis
PASS: [IC] Erythrose 4-phosphate de novo synthesis
PASS: [IC] Fructose 6-phosphate de novo synthesis
PASS: [IC] Glyceraldehyde 3-phosphate de novo synthesis
PASS: [IC] Glucose 6-phosphate de novo synthesis
PASS: [IC] Mitochondrial oxaloacetate de novo synthesis
PASS: [IC] Phosphoenolpyruvate de novo synthesis
PASS: [IC] Pyruvate de novo synthesis
PASS: [IC] Ribose 5-phosphate de novo synthesis
PASS: [IC] Mitochondrial succinyl-CoA de novo synthesis
PASS: [BS] Cholesterol de novo synthesis
PASS: [BS] Protein synthesis from AAs
PASS: [ER] Oxidative phosphorylation
PASS: [ER] Oxidative decarboxylation
PASS: [ER] Krebs cycle NADH
PASS: [ER] Ubiquinol-to-proton
PASS: [ER] Ubiquinol-to-ATP
PASS: [SU] Beta oxidation of saturated FA
PASS: [SU] Beta oxidation of long-chain FA
```

I will appreciate it if someone gives me advice regarding this issue.

**Comment by mohammadrezazargar** [comment]
Hi. I have the same problem. It's now more than one day that it is in this step and didn't go further.

**Comment by JonathanRob** [comment]
Hi @mohammadrezazargar, is your script getting stuck at the same task (beta oxidation of long-chain FA)?

**Reply by mohammadrezazargar** [reply]
yes. Its now about 2 days.
<img width="702" alt="Screenshot 2025-02-25 at 10 46 39" src="https://github.com/user-attachments/assets/9e59ea1a-4e42-4260-b7c9-625c5bbedfa6" />

**Comment by JonathanRob** [comment]
Ok, it definitely should only take on the order of minutes, so something is definitely going wrong. Perhaps it is a solver issue?

Sorry I cannot test/debug it myself, since I no longer have a Matlab license. Maybe @edkerk or @johan-gson are able to take a look?

**Comment by IVANDOMENZAIN** [comment]
Having the same problem, gets stuck at exactly the same task @JonathanRob @sangmok920622 @mohammadrezazargar, agree that it might be related to solver settings. In my case I am using `gurobi 11.0.2` and standard `RAVEN` settings. Gonna try to play around with solver settings and also see what happens if I temporarily remove such task from the list, just to see if other problems arise.
